scripts/approve-fingerprint-sudo.sh

```diff
@@ -5,10 +5,19 @@
 # To be able to use the mac touchbar for authorizing the 'sudo' command in terminal
 # This will persist through software updates unlike changes directly made to '/etc/pam.d/sudo'
 # Copied from: https://apple.stackexchange.com/a/466029
-# TODO: Need to ensure that TouchId hardware is present before running this script
 
 type is_file &> /dev/null 2>&1 || source "${HOME}/.shellrc"
 
+function check_touchid_hardware() {
```

Comment:
In all of the scripts, I have not been using the keyword `function`. Please remove it

Reply:
Also, since this is not used anywhere else, is there are need to make it a function?